src/components/OperationTile/TransactionTile.tsx

```diff
@@ -15,21 +15,30 @@ import { TransactionOperation } from "../../utils/tezos";
 import { AddressPill } from "../AddressPill/AddressPill";
 
 export const TransactionTile: React.FC<{ operation: TransactionOperation }> = ({ operation }) => {
-  const isOutgoing = useIsOutgoingOperation(operation.sender.address);
+  const operationDestination = useGetOperationDestination(
+    operation.sender.address,
+    operation.target.address
+  );
   const amount = prettyTezAmount(String(operation.amount));
   const showToAddress = useShowAddress(operation.target.address);
   const showFromAddress = useShowAddress(operation.sender.address);
   // if you send assets between your own accounts you need to see at least one address
   const showAnyAddress = !showToAddress && !showFromAddress;
 
-  const titleColor = isOutgoing ? colors.orange : colors.green;
-  const sign = isOutgoing ? "-" : "+";
+  const titleColor =
```

Comment:
nit: code for determining color & sign is duplicated. Maybe we can move it to operation destination?